joomla/joomla-cms
# [4.0] Clean up Bootstrap client side

Pull Request for Issue # .

### Summary of Changes
- No inline scripts
- A redo of #11922 (was RTC, before falling out of sync)

### Testing Instructions
Apply patch, ensure that tooltips, alerts, modals, tabs accordions etc still work

### Expected result



### Actual result



### Documentation Changes Required
Nope, internal changes
